### ISSUE TITLE
make migrate throws error in SQL

### ISSUE THREAD

**Issue by emergingdzns** [systemsdk/docker-nginx-php-laravel#41]
I just set this up and am following the directions. I'm getting an error when attempting to run `make migrate`. It's giving me this error:
```
SQLSTATE[HY000] [2002] No such file or directory (SQL: select * from information_schema.tables where table_schema = laravel and table_name = migrations and table_type = 'BASE TABLE')
```

I checked the `information_schema` database `tables` table and the row is in fact there. The migrations table is in fact in the database.

I can't seem to get past this. Any ideas?

I can connect to the database using my SequelAce app from my local system.

**Comment by emergingdzns** [comment]
I've also changed the host from 127.0.0.1 to localhost. Neither has made a difference.

**Comment by dimadeush** [comment]
We are using host as "mysql" and it is working properly inside CI and on local PC. Maybe you have changed something on your local PC or environment. Also please make sure that all docker containers are running on your local PC.

**Comment by emergingdzns** [comment]
Thanks @dimadeush but I've confirmed it is running. In fact i can connect to it via the port 33061 from my Mac. I kept the ports defined in the yml files provided in the repo as `- 33061:3306`.

It is part of the `laravel` network as well.

In fact the initial run even performs the mysql initdb.sql queries I have in there to build the database and basic table schemas.

One thing I noticed that's different is that nginx, laravel, supervisor and my redis configs all say `image: ${COMPOSE_PROJECT_NAME}-xxxx:latest` (where xxxx is the name of the config). But the mysql entry just says mysql:8.0. I tried adding the `${COMPOSE_PROJECT_NAME}-` prefix but it threw an error so undid that.

It's acting like the migration is trying to use a login that isn't correct (like not the root user). But my env shows:
```
DB_HOST=localhost
DB_DATABASE=laravel
DB_USERNAME=root
DB_PASSWORD=secret
```

That's the data I connect to the mysql db with on my mac so I know those credentials work. 

I'm stumped.
